source/tutorial/simple-component.md

```diff
@@ -312,63 +318,66 @@ In the first test, we just want to verify the output of the component, so we jus
 ```tests/integration/components/rental-listing-test.js{+4,+5}
 test('should display rental details', function(assert) {
   this.set('rentalObj', rental);
-  this.render(hbs`{{rental-listing rental=rentalObj}}`);
+  await render(hbs`{{rental-listing rental=rentalObj}}`);
   assert.equal(this.$('.listing h3').text(), 'test-title', 'Title: test-title');
```

Comment:
This probably should not use jquery anymore

Reply:
yes.  in the process of killing all the jqueries 👍